### ISSUE TITLE
Need help with uninstalling supplemental elements with NodeJS

### ISSUE THREAD

**Issue by MCM-Math** [nodejs/help#2475]
I am currently taking an online course that includes learning NodeJS, so I installed NodeJS on my computer, and I clicked on the option to install Chocolatey and other related things.  However, I am not sure how to uninstall these things, and Chocolatey says its uninstallation script "may destroy your machine and require a rebuild. It may have varied results on different machines in the same environment. Think twice before running this."  This is not a confidence builder for their product for me.

I didn't catch all the text that scrolled by during the installation process, and the window closed when I tried to click on it to copy the text.  I would like to know how to make sure to uninstall all of this extra nonsense, as well as anything else that was installed with NodeJS.

Basically, I would like to know how to completely uninstall not just NodeJS, but everything else like Chocolatey and all the packages Chocolatey installed on its own, and all the other things in the NodeJS installation.

**Comment by Hakerh400** [comment]
See https://github.com/nodejs/help/issues/2177#issuecomment-530261546

**Comment by PoojaDurgad** [comment]
@MCM-Math - as per the below link which will install the additional unwanted tools 
  - https://github.com/nodejs/node/blob/master/tools/msvs/install_tools/install_tools.bat   
probably with your issue you could uninstall the unwanted tools manually in the windows setting->apps and features.

**Comment by PoojaDurgad** [comment]
inactive. closing !! feel free to reopen the issue if that is not the case.
